tests/test_signals.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.auth import signals
+from django.test import TestCase
+from django.test.client import RequestFactory
+from fd_dj_accounts.models import User
+
+
+class SignalTestCase(TestCase):
+
+    def test_update_last_login(self) -> None:
+        """Only `last_login` is updated in `update_last_login`"""
+        user = User.objects.create_user(email_address='staff@a.com', password='password')
+        request = RequestFactory().get('/login')
+        signals.user_logged_in.send(sender=user.__class__, request=request, user=user)
+        old_last_login = user.last_login
```

Comment:
Check that the signal adds a new datetime

```suggestion
        self.assertIsNotNone(user.last_login)
        old_last_login = user.last_login
```